subgraph/src/utils/helper.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-underscore-dangle */
+import { BigInt as GraphBN, Bytes, ethereum } from "@graphprotocol/graph-ts";
+
+import { Account, MACI, User } from "../../generated/schema";
+
+import { DEFAULT_MACI_ID } from "./constants";
+
+export const packPubkey = (pubkeyX: GraphBN, pubkeyY: GraphBN): string => `${pubkeyX.toString()}-${pubkeyY.toString()}`;
+
+export const createOrLoadMACI = (
+  event: ethereum.Event,
+  owner: Bytes | null = null,
+  stateTreeDepth: GraphBN = GraphBN.fromI32(10),
+): MACI => {
+  let maci = MACI.load(DEFAULT_MACI_ID);
+
+  if (!maci) {
+    maci = new MACI(DEFAULT_MACI_ID);
+    maci.owner = owner !== null ? owner : (event.transaction.from as Bytes);
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Here is the same issue, maybe it can be cast as the line above (with as Bytes but here with as I32)

Reply:
I've changed those to bigint for easier work around, thank you